etc/config/cuda.amazon.properties

```diff
@@ -61,7 +61,21 @@ libs.cueigen.versions.trunk.path=/opt/compiler-explorer/libs/eigen/vtrunk
 libs.cueigen.versions.334.version=3.3.4
 libs.cueigen.versions.334.path=/opt/compiler-explorer/libs/eigen/v3.3.4
 
-libs.cucub.name=CUB
+libs.thrustcub.name=Thrust+CUB
+libs.thrustcub.versions=trunk:109090:109100:109101:110000
+libs.thrustcub.url=http://www.github.com/NVIDIA/thrust
+libs.thrustcub.versions.109090.version=1.9.9
+libs.thrustcub.versions.109090.path=/opt/compiler-explorer/libs/thrustcub/1.9.9:/opt/compiler-explorer/libs/thrustcub/1.9.9/dependencies/cub
+libs.thrustcub.versions.109100.version=1.9.10
+libs.thrustcub.versions.109100.path=/opt/compiler-explorer/libs/thrustcub/1.9.10:/opt/compiler-explorer/libs/thrustcub/1.9.10/dependencies/cub
+libs.thrustcub.versions.109101.version=1.9.10-1
+libs.thrustcub.versions.109101.path=/opt/compiler-explorer/libs/thrustcub/1.9.10-1:/opt/compiler-explorer/libs/thrustcub/1.9.10-1/dependencies/cub
+libs.thrustcub.versions.110000.version=1.10.0
+libs.thrustcub.versions.110000.path=/opt/compiler-explorer/libs/thrustcub/1.10.0:/opt/compiler-explorer/libs/thrustcub/1.10.0/dependencies/cub
+libs.thrustcub.versions.trunk.version=trunk
+libs.thrustcub.versions.trunk.path=/opt/compiler-explorer/libs/thrustcub-nightly/trunk:/opt/compiler-explorer/libs/thrustcub-nightly/1.10.0/dependencies/cub
```

Comment:
Can you set this to /opt/compiler-explorer/libs/thrustcub/trunk instead, I'll fix the infra yaml to reflect that

Reply:
Absolutely. I wanted to set things up that way originally, but wasn't sure how to handle it in infra. 👍 Fixed in last push.

Reply:
For reference, this way: https://github.com/compiler-explorer/infra/commit/86d101ccaddbae1148bd485485205019a43dfd4d
(Can be the same name as long as it's under a different parentkey)